chain/consensus/mir/state_manager.go

```diff
@@ -152,8 +153,14 @@ func (sm *StateManager) syncFromPeers(tsk types.TipSetKey) (err error) {
 	attempt := time.NewTicker(PeerDiscoveryInterval)
 	defer attempt.Stop()
 
+	b := backoff.Backoff{
```

Comment:
Do we really need this? We are introducing an external library that it is unclear its level of maintainence for something that we are kind of handling by the dynamic timeout you introduced in line 711. I would try to live without this if possible

Reply:
Removed